### DISCUSSION TITLE
TTN Mapper Configuration | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1655] [Q&A]
# TTN Mapper Configuration | Seeed Studio Wiki

SenseCAP_Tracker_T1000-A/B_TTN_Mapper

https://wiki.seeedstudio.com/ttn_mapper_for_SenseCAP_T1000/

**Comment by Kfagermo** [comment]
Can we also have one that covers how to do the helium mappers?
